### PR TITLE
Fix Sqreen README

### PR DESCRIPTION
**Pull request by yzhan289**
Links aren't working when using `()`

### What does this PR do?

A brief description of the change being made with this pull request.

### Motivation

What inspired you to submit this pull request?

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

**Review by ChristineTChen — APPROVED** [review]

File: sqreen/README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Sqreen streams your services security activity to Datadog as logs.
 
 To set it up:
 
-1. Go to the [Sqreen Dashboard > Organization settings > Integrations](3).
+1. Go to the [Sqreen Dashboard > Organization settings > Integrations][3].
 2. Connect a new Datadog accounts
```
Comment (alai97) [inline sqreen/README.md:32]:
```suggestion
2. Connect a new Datadog account.
```

File: sqreen/README.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ Sqreen streams your services security activity to Datadog as logs.
...
 3. Choose the corresponding intake server URL. For Datadog US: `http-intake.logs.datadoghq.com`. For Datadog EU: `http-intake.logs.datadoghq.eu`.
```
Comment (alai97) [inline sqreen/README.md:33]:
```suggestion
3. Choose the corresponding intake server URL. For Datadog US, use `http-intake.logs.datadoghq.com`. For Datadog EU, use `http-intake.logs.datadoghq.eu`.
```

File: sqreen/README.md [hunk 3]
```diff
@@ -28,7 +28,7 @@ Sqreen streams your services security activity to Datadog as logs.
...
 4. In the "API Key" field, provide a [Datadog API key][4] used to stream the Sqreen data as logs.
```
Comment (alai97) [inline sqreen/README.md:34]:
```suggestion
4. In the **API Key** field, provide a [Datadog API key][4] which streams the Sqreen data as logs.
```

File: sqreen/README.md [hunk 4]
```diff
@@ -45,7 +45,7 @@ For support, reach out to [Datadog Support](/help).
 
 ## Payload reference documentation
```
Comment (alai97) [inline sqreen/README.md:46]:
```suggestion
## Payload reference
```

**Review by alai97 — APPROVED** [review]
Thanks for fixing the links, @yzhan289! If you don't mind, I included some minor copy suggestions as well.

**Pull request merged**